conda-recipe/rsmtool/meta.yaml

```diff
@@ -9,7 +11,7 @@ build:
   number: 0
   script:
     - cd $SRC_DIR
-    - $PYTHON setup.py install
+    - $PYTHON setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
I think you don't need to do this since its use is discouraged. See the SKLL recipe for a much easier alternative.